apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerBatchCommand.ExecutableCommand.cs

```diff
@@ -51,6 +54,8 @@ public ExecutableCommand(SpannerBatchCommand command)
                 CommandType = command.CommandType;
                 Priority = command.Priority;
                 Tag = command.Tag;
+                _lazyConversionOptions = new Lazy<SpannerConversionOptions>(
```

Comment:
Sorry, I wasn't suggesting using `Lazy<T>` here - I was only suggesting that we evaluate the ConversionOptions property as a local variable before the foreach loop around line 102. Even that is probably premature optimization, to be honest. I think we should probably just go back to making the property call SpannerConversionOptions.ForConnection.

Reply:
My apologies for interpreting it incorrectly. I have now reverted the `Lazy<T>` change.